.github/workflows/create-release-tag.yaml

```diff
@@ -0,0 +1,60 @@
+name: tag-generation
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+    paths:
+      - 'CHANGELOG.md'
+
+jobs:
+  create-tag:
+    if: github.event.pull_request.merged == true && contains(join(github.event.pull_request.labels.*.name, ','), 'release') && startsWith(github.event.pull_request.title, 'Update changelog to prep for release')
```

Comment:
```suggestion
    if: github.event.pull_request.merged == true && contains(join(github.event.pull_request.labels.*.name, ','), 'release') && startsWith(github.event.pull_request.title, 'release: update changelog to prep for release')
```

Reply:
We will have to change the way the PR script is generating the title. The current way the title actually ends up being "Update changelog to prep for release `x.x.x`". Are you suggesting we make the title "release: update changelog `x.x.x`"? It's not as semantic.